styles/components/_current-projects.scss

```diff
@@ -0,0 +1,145 @@
+.cmp-current-projects {
+  $full-width: 100%;
+  $font-weight-light: 200;
+  $font-weight-regular: 400;
+  $font-weight-semibold: 600;
+  $tablet-screen-width: 60rem;
+  $mobile-screen-width: 40rem;
+
+  box-sizing: border-box;
+  background-color: color("gray-400");
+  padding: space("2xl");
+  font-size: font-size("lg");
+
+  &__header {
+    margin: 0;
+    margin-bottom: space("lg");
+    font-family: font-family("sans-extended");
+    font-size: font-size("xl");
+    font-weight: $font-weight-semibold;
+    letter-spacing: -0.02em;
+    text-align: left;
+
+    &--caption {
+      @media (max-width: $tablet-screen-width) {
```

Comment:
I wouldn't point this out if I didn't know it's in your well within your capabilities to rework this, but if you look at other sparkbox projects the preferred approach to media queries is `min-width` not `max-width` so I think it would be a good idea if we stuck with `min-width` on this project. If there is a good reason why it the media queries need to be `max-width` here just let me know and I can un-request changes